externals/kyuubi-flink-sql-engine/pom.xml

```diff
@@ -61,19 +61,19 @@
 
         <dependency>
             <groupId>org.apache.flink</groupId>
-            <artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
+            <artifactId>flink-streaming-java</artifactId>
```

Comment:
Based on the discussion, Kyuubi should support 2 or 3 Flink versions, this change will break support for Flink 1.14

Reply:
That's indeed a critical issue. Kyuubi uses some non-public Flink APIs, which have no backward compatibility guarantee. We may need a multi-version solution like FlinkShims in Zeeplin. 

@wolfboys Does StreamX use Flink internal APIs as well? How does it achieve multi-version support? Thanks!

Reply:
> That's indeed a critical issue. Kyuubi uses some non-public Flink APIs, which have no backward compatibility guarantee. We may need a multi-version solution like FlinkShims in Zeeplin.
> 
> @wolfboys Does StreamX use Flink internal APIs as well? How does it achieve multi-version support? Thanks!

Since I am not familiar with the Kyuubi project, I originally wanted to upgrade the version to Flink 1.15, implement this, and then consider multi version support. Now it seems that upgrading directly to 1.15 will affect the support of the released Flink 1.14. I will bring up the PR supported by multiple versions later